Lib/unittest/mock.py

```diff
@@ -3003,6 +3005,98 @@ def __set__(self, obj, val):
         self(val)
 
 
+_timeout_unset = sentinel.TIMEOUT_UNSET
+
+class ThreadingMixin(Base):
+
+    DEFAULT_TIMEOUT = None
+
+    def _get_child_mock(self, /, **kw):
+        if "timeout" in kw:
+            kw["timeout"] = kw.pop("timeout")
```

Comment:
@mariocj89 - this path has no test coverage:
https://app.circleci.com/pipelines/github/testing-cabal/mock/1428/workflows/7b4dab2f-46de-4dd7-bb8b-5d2000975547/jobs/20351

Please can you PR to add some? (and feel free to do the ones from @graingert's [PR](https://github.com/python/cpython/pull/94962) if you have energy ;-)

Reply:
Good catch, this is indeed no longer used as we moved to `_mock_wait_timeout`

Reply:
https://github.com/python/cpython/pull/106591